code/stable/nopcm/src/cpp/Control.cpp

```diff
@@ -23,7 +23,7 @@ int main(int argc, const char *argv[]) {
     string inputfile = argv[1];
     InputParameters inParams = InputParameters();
     func_get_input(inputfile, inParams);
-    input_constraints(inParams);
+    input_constraints(A_C, C_W, h_C, T_init, t_final, L, T_C, t_step, rho_W, D, T_W, E_W);
```

Comment:
Does this compile? The previous and next line uses `inParams` and here they are all exploded.